homeassistant/components/vicare/binary_sensor.py

```diff
@@ -151,6 +163,16 @@ async def async_setup_entry(
         if entity is not None:
             entities.append(entity)
 
+    for description in GLOBAL_SENSORS:
+        entity = _build_entity(
+            f"{name} {description.name}",
+            api,
+            hass.data[DOMAIN][config_entry.entry_id][VICARE_DEVICE_CONFIG],
+            description,
+        )
+        if entity is not None:
+            entities.append(entity)
+
```

Comment:
Are you not duplicating functionnality here with what is above on lines 155-164?